.github/workflows/pr.yaml

```diff
@@ -1,13 +1,29 @@
-name: Test Operator
+# Copyright 2023 The Janus IDP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: PR Test operator image
```

Comment:
I do not think we deal with image here, so could it be just "PR Test [Operator]" ?

Reply:
ah, good point -- it WAS building the image (with both docker and podman) but you removed that.